lens.cabal

```diff
@@ -348,6 +335,10 @@ library
 
   default-language: Haskell2010
 
+  -- future proof, whether the field will be comma separated or not.
+  x-docspec-extra-packages: simple-reflect
+  x-docspec-extra-packages: deepseq
```

Comment:
I'm a bit confused—`deepseq` is listed as an `x-docspec-extra-package` here, but it's not listed in the `build-depends` stanza of the `doctests` test suite. What is the reason for this discrepancy?

Reply:
You wanted less duplication :)

Reply:
To be clear, my dream is not to have any duplication at all, but if I understand https://github.com/ekmett/lens/pull/958#discussion_r549519356 correctly, that isn't feasible unless `cabal-install` gains some additional functionality. Given that, I reluctantly accept the fact that we need to duplicate some information for the time being.

But the state that this `.cabal` file is in now is an uncomfortable medium between no duplication and duplicating every `x-docspec-extra-package`, as we duplicate one of these packages (`simple-reflect`) but not the other (`deepseq`). This is even more mysterious to me, and when I inevitably revisit this code months later, I'm probably not going to remember the subtleties that allow this to work. I would request that we either:

* Just duplicate both `simple-reflect` and `deepseq`, to make it more visually obvious that these parts of the `.cabal` file are meant to be in sync, or
* Leave a comment somewhere explaining why `deepseq` is omitted.

Reply:
I duplicated both, and added a comment why we need `doctests`. And what needs to happen, so it can be removed.